rafthttp/peer.go

```diff
@@ -31,6 +31,18 @@ const (
 	ConnWriteTimeout = 5 * time.Second
 
 	recvBufSize = 4096
+
+	streamApp   = "streamMsgApp"
+	streamMsg   = "streamMsg"
+	pipelineMsg = "pipeline"
```

Comment:
the next step is to merge them (there are also something i want to refactor in stream.go). but not in this pr.